### PR TITLE
New release 1.1.3

### PR DESCRIPTION
**Pull request by maorlipchuk**

### CODE REVIEW

**Comment by maorlipchuk** [comment]
ci please build

**Comment by maorlipchuk** [comment]
ci please build

**Comment by maorlipchuk** [comment]
ci please build

File: build.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 VERSION="1.1.3"
-MILESTONE=master
+MILESTONE=
 RPM_RELEASE="1.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
```
Comment (mwperina) [inline build.sh]: Shouldn't the RPM release be "1" as this is official release?

Comment (maorlipchuk) [reply build.sh]: sure, done

**Comment by maorlipchuk** [comment]
ci please build

**Pull request merged**